lambda_backend/sns_setup/src/telegramDispatcher.ts

```diff
@@ -0,0 +1,27 @@
+import { Handler, SNSEvent } from 'aws-lambda';
+
+import axios from 'axios';
+
+export const handler: Handler = async (event: SNSEvent) => {
+  if (process.env['BOT_TOKEN'] === '' || process.env['BOT_CHAT_ID'] === '') {
+    console.error('BOT_TOKEN or BOT_CHAT_ID not set');
```

Comment:
Should this immediately return some error statusCode instead of continuing to execute the function?